ios/Classes/DarwinAudioSession.m

```diff
@@ -407,6 +411,28 @@ - (void) audioInterrupt:(NSNotification*)notification {
     }
 }
 
+- (void) routeChange:(NSNotification*)notification {
+    NSNumber *routeChangeReasonType = (NSNumber*)[notification.userInfo valueForKey:AVAudioSessionRouteChangeReasonKey];
+    NSLog(@"routeChange detected");
+    [self invokeMethod:@"onRouteChange" arguments:@[@([routeChangeReasonType integerValue])]];
```

Comment:
Let's pass a cut-down fragment of the route description here as a second argument